changelog.md

```diff
@@ -1,4 +1,33 @@
-# MultiMC 0.6.16
+# MultiMC 0.7.0
+
+This is a nice big rollup release with all kinds of things that accumulated since the last release.
+
+With this, we are also abandoning the concept of stable releases. From now on, MultiMC will always use the `develop` channel.
```

Comment:
Why do you always need to have an "develop" channel when a old stable release I have (now develop).